src/lightning_app/utilities/exceptions.py

```diff
@@ -41,6 +41,14 @@ class LightningWorkException(Exception):
     """Exception used to inform users of misuse with LightningWork."""
 
 
+class LightningPlatformException(Exception):  # pragma: no cover
```

Comment:
Where is this used?

Reply:
@Borda On the side of the launcher (platform) 
The idea is to raise it if something happens on the platform that the framework should (could) react to (e.g. cannot start work due to out of credits)